Header header
float64 utc_time
float64 yaw_deg
float64 tilt_deg
float64 range    # meters
int8 gps_quality # 0-4
float64 pdop     # Precision dilution of precision
int8 sats_used 
